### PR TITLE
[ALLUXIO-1749] Make thrift largest frame size adjustable

### PR DESCRIPTION
**Pull request by pfxuan**
Thrift v0.9.3 changes the default size of largest allowable frame from 1GB (v0.9.3) to 16MB. It could become an issue in current implementation. This PR is used to make largest frame size adjustable and help further optimizition.

### CODE REVIEW

File: core/common/src/main/java/alluxio/security/authentication/AuthenticationUtils.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ public static TTransportFactory getServerTransportFactory(Configuration conf)
     AuthType authType = conf.getEnum(Constants.SECURITY_AUTHENTICATION_TYPE, AuthType.class);
     switch (authType) {
       case NOSASL:
-        return new TFramedTransport.Factory();
+        return new TFramedTransport.Factory(conf.getInt(Constants.THRIFT_FRAME_SIZE_BYTES_MAX));
```
Comment (jsimsa) [inline core/common/src/main/java/alluxio/security/authentication/AuthenticationUtils.java]: use getBytes here

File: core/common/src/main/resources/alluxio-default.properties [hunk 2]
```diff
@@ -24,6 +24,7 @@ alluxio.keyvalue.partition.size.bytes.max=512MB
 alluxio.metrics.conf.file=${alluxio.home}/conf/metrics.properties
 alluxio.network.host.resolution.timeout.ms=5000
 alluxio.test.mode=false
+alluxio.thrift.frame.size.bytes.max=67108864
```
Comment (jsimsa) [inline core/common/src/main/resources/alluxio-default.properties]: use 64MB here

File: docs/_data/table/common-configuration.csv [hunk 3]
```diff
@@ -7,6 +7,7 @@ alluxio.keyvalue.partition.size.bytes.max,512MB
 alluxio.metrics.conf.file,${alluxio.home}/conf/metrics.properties
 alluxio.network.host.resolution.&#8203;timeout.ms,5000
 alluxio.test.mode,false
+alluxio.thrift.frame.size.bytes.max,67108864
```
Comment (jsimsa) [inline docs/_data/table/common-configuration.csv]: 64MB here

**Comment by jsimsa** [comment]
@pfxuan thanks for the PR, left some minor comments

**Comment by pfxuan** [comment]
@jsimsa, the comments have been addressed. Please help check.

**Comment by jsimsa** [comment]
LGTM

**Pull request merged**